IntegrationTests/UI.IntegrationTests/CommandsDialogs/FormBrowseTests.cs

```diff
@@ -184,6 +184,44 @@ public void Filters_should_behave_as_expected()
                 });
         }
 
+        [TestCase("", "file.txt")]
+        [TestCase("", "file with spaces.txt")]
```

Comment:
```suggestion
        [TestCase("", "file with spaces.txt")]
        [TestCase("", "\"file with spaces.txt\"")]
```

Reply:
...actually we can't do that
```
System.IO.IOException : The filename, directory name, or volume label syntax is incorrect. : 'C:\Users\appveyor\AppData\Local\Temp\1\tqbhkuu3.lkg\repo1\"file with spaces.txt"'
 [463](https://ci.appveyor.com/project/gitextensions/gitextensions/build/4.0.2.15936#L463)   at Microsoft.Win32.SafeHandles.SafeFileHandle.CreateFile(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options)
 [464](https://ci.appveyor.com/project/gitextensions/gitextensions/build/4.0.2.15936#L464)   at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize)
 [465](https://ci.appveyor.com/project/gitextensions/gitextensions/build/4.0.2.15936#L465)   at System.IO.Strategies.OSFileStreamStrategy..ctor(String path, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize)
 [466](https://ci.appveyor.com/project/gitextensions/gitextensions/build/4.0.2.15936#L466)   at System.IO.StreamWriter.ValidateArgsAndOpenPath(String path, Boolean append, Encoding encoding, Int32 bufferSize)
 [467](https://ci.appveyor.com/project/gitextensions/gitextensions/build/4.0.2.15936#L467)   at System.IO.File.WriteAllText(String path, String contents)
 [468](https://ci.appveyor.com/project/gitextensions/gitextensions/build/4.0.2.15936#L468)   at CommonTestUtils.GitModuleTestHelper.CreateFile(String parentDir, String fileName, String fileContent) in C:\projects\gitextensions\UnitTests\CommonTestUtils\GitModuleTestHelper.cs:line 77
 [469](https://ci.appveyor.com/project/gitextensions/gitextensions/build/4.0.2.15936#L469)   at CommonTestUtils.GitModuleTestHelper.CreateRepoFile(String fileRelativePath, String fileName, String fileContent) in C:\projects\gitextensions\UnitTests\CommonTestUtils\GitModuleTestHelper.cs:line 99
 [470](https://ci.appveyor.com/project/gitextensions/gitextensions/build/4.0.2.15936#L470)   at CommonTestUtils.ReferenceRepository.CreateCommitRelative(String fileRelativePath, String fileName, String commitMessage, String content) in C:\projects\gitextensions\UnitTests\CommonTestUtils\ReferenceRepository.cs:line 97
 [471](https://ci.appveyor.com/project/gitextensions/gitextensions/build/4.0.2.15936#L471)   at GitExtensions.UITests.CommandsDialogs.FormBrowseTests.File_history_should_behave_as_expected(String fileRelativePath, String fileName) in C:\projects\gitextensions\IntegrationTests\UI.IntegrationTests\CommandsDialogs\FormBrowseTests.cs:line 197
```

Reply:
There should be a testcase that quotes the complete path, Add a parameter?

Reply:
Added a test for the main dialog. The same test can't apply to the History dialog - it leads to exceptions. I have no plans to investigate any further.